Snakefile

```diff
@@ -25,143 +25,24 @@ rule k51:
-    wget -O {output} https://osf.io/g94n5/download
-    '''
-
-rule download_genbank_archaea_zip_k31:
-    output: "inputs/sourmash_databases/genbank-2022.03-archaea-k31.zip"
-    conda: "envs/wget.yml"
-    shell:'''
-    wget -O {output} https://osf.io/hfybv/download
-    '''
-
-rule download_genbank_archaea_zip_k51:
-    output: "inputs/sourmash_databases/genbank-2022.03-archaea-k51.zip"
-    conda: "envs/wget.yml"
-    shell:'''
-    wget -O {output} https://osf.io/dehrc/download
-    '''
-
-rule download_genbank_archaea_lineage:
-    output: "inputs/sourmash_databases/genbank-2022.03-archaea.lineages.csv.gz"
-    conda: "envs/wget.yml"
-    shell:'''
-    wget -O {output} https://osf.io/kcbpn/download
-    '''
-
-rule download_genbank_viral_zip_k21:
-    output: "inputs/sourmash_databases/genbank-2022.03-viral-k21.zip"
-    conda: "envs/wget.yml"
-    shell:'''
-    wget -O {output} https://osf.io/updvc/download
-    '''
-
-rule download_genbank_viral_zip_k31:
-    output: "inputs/sourmash_databases/genbank-2022.03-viral-k31.zip"
-    conda: "envs/wget.yml"
-    shell:'''
-    wget -O {output} https://osf.io/egkt2/download
-    '''
-
-rule download_genbank_viral_zip_k51:
-    output: "inputs/sourmash_databases/genbank-2022.03-viral-k51.zip"
-    conda: "envs/wget.yml"
-    shell:'''
-    wget -O {output} https://osf.io/z8scg/download
-    '''
-
-rule download_genbank_viral_lineage:
-    output: "inputs/sourmash_databases/genbank-2022.03-viral.lineages.csv.gz"
-    conda: "envs/wget.yml"
-    shell:'''
-    wget -O {output} https://osf.io/j4tsu/download
-    '''
-
-rule download_genbank_protozoa_zip_k21:
-    output: "inputs/sourmash_databases/genbank-2022.03-protozoa-k21.zip"
-    conda: "envs/wget.yml"
-    shell:'''
-    wget -O {output} https://osf.io/m23r6/download 
-    '''
-
-rule download_genbank_protozoa_zip_k31:
-    output: "inputs/sourmash_databases/genbank-2022.03-protozoa-k31.zip"
-    conda: "envs/wget.yml"
-    shell:''' 
-    wget -O {output} https://osf.io/zm5vg/download
-    '''
-
-rule download_genbank_protozoa_zip_k51:
-    output: "inputs/sourmash_databases/genbank-2022.03-protozoa-k51.zip"
-    conda: "envs/wget.yml"
-    shell:'''
-    wget -O {output} https://osf.io/32y98/download
-    '''
-
-rule download_genbank_protist_lineage:
-    output: "inputs/sourmash_databases/genbank-2022.03-protozoa.lineages.csv.gz"
-    conda: "envs/wget.yml"
-    shell:'''
-    wget -O {output} https://osf.io/2x8u4/download
-    '''
+rule download_sourmash_databases_genbank:
+    input: "inputs/sourmash_databases/sourmash-database-info.csv"
+    output: "inputs/sourmash_databases/genbank-2022.03-{lineage}-k{ksize}.zip"
+    run:
+        sourmash_database_info = pd.read_csv(str(input[0]))
+        ksize = int(wildcards.ksize)
+        lineage_df = sourmash_database_info.loc[(sourmash_database_info['lineage'] == wildcards.lineage) & (sourmash_database_info['ksize'] == ksize)]
+        osf_hash = lineage_df['osf_hash'].values[0] 
```

Comment:
It'd be good to check that `lineage_df` is defined and is not `None` or smth. And alert the user if it's not defined.